src/services/api.js

```diff
@@ -1,7 +1,15 @@
+const baseUrl = 'https://api.mercadolibre.com/sites/MLB';
+
 export async function getCategories() {
-  // Implemente aqui
+  const response = await fetch(`${baseUrl}/categories`);
+  const categories = await response.json();
+
+  return categories;
 }
 
-export async function getProductsFromCategoryAndQuery(/* categoryId, query */) {
-  // Implemente aqui! Quando o fizer, descomente os parâmetros que essa função recebe
+export async function getProductsFromCategoryAndQuery(categoryId, query) {
+  const response = await fetch(`${baseUrl}/search?q=${query}+category=${categoryId}`);
```

Comment:
```suggestion
  const response = await fetch(`${baseUrl}/search?q=${query}&category=${categoryId}`);
```
Boa tarde, nesta busca se você colocar o + após o query, ele entende que o `category=${categoryId}` ainda faz parte da busca, para resolver o problema seria interessante trocar pelo &, pois é ele que faz a separação dos itens da busca.